documentation/01-Prerequisites.md

```diff
@@ -109,12 +105,11 @@ docker build --build-arg=UbuntuVersion=$UbuntuVersion -t mlos/build:ubuntu-$Ubun
 ```
 
 > Where `UbuntuVersion` can also be set to another supported version of Ubuntu.
-
 > Tip: you can also pass `--build-arg=http_proxy=http:/some-proxy-caching-host:3128` to help direct `apt` and `pip` to fetch the necessary packages via local caches.
 
 See [02-Build.md](./02-Build.md#docker) for instructions on how to run this image.
 
-#### Manual Build Tools Install
+### Option 2: Manual Build Tools Install
```

Comment:
:thumbsup: I like the `Option N` headers

Reply:
There may be links to these sections from other places, but I will try and clean those up in my other dead links cleanup PR.